packages/oauth/oauth-provider/src/lib/html/html.ts

```diff
@@ -49,11 +49,7 @@ export class Html implements Iterable<string> {
 
   *[Symbol.iterator](): IterableIterator<string> {
     for (const fragment of this.#fragments) {
-      if (typeof fragment === 'string') {
-        yield fragment
-      } else {
-        yield* fragment
-      }
+      yield String(fragment)
```

Comment:
This change enables the `toString()` optimization from this class to be used. For some reason (which I did not take time to debug any further), the previous behavior could cause this `HTML` instance to become "empty" after it was read once.